integration_tests/test_poller_integration.py

```diff
@@ -630,3 +630,114 @@ def run_retried_single_search(setup_splunk, search_string, retries):
         logger.info("No results returned from search. Retrying in 2 seconds...")
         time.sleep(2)
     return 0, 0
+
+
+@pytest.fixture(scope="class")
+def setup_single_condition_profiles(request):
+    trap_external_ip = request.config.getoption("trap_external_ip")
+    profiles = {
+        "gt_profile": {
+            "frequency": 7,
+            "varBinds": [yaml_escape_list(sq("IF-MIB"), sq("ifOutDiscards"))],
+            "conditions": [
+                {"field": "SNMPv2-MIB.sysORUpTime", "operation": "gt", "value": 3}
+            ],
+        },
+        "lt_profile": {
+            "frequency": 7,
+            "varBinds": [yaml_escape_list(sq("IF-MIB"), sq("ifOutDiscards"))],
+            "conditions": [
+                {"field": "SNMPv2-MIB.sysORUpTime", "operation": "lt", "value": 3}
+            ],
+        },
+        "in_profile": {
+            "frequency": 7,
+            "varBinds": [yaml_escape_list(sq("IF-MIB"), sq("ifOutDiscards"))],
+            "conditions": [
+                {
+                    "field": "SNMPv2-MIB.sysORDesc",
```

Comment:
Conditional profiles elect the SNMP objects that will be polled, so the conditions and varBinds should be part of the same MIB family. We cannot check the description of the system in order to include or exclude the whole device (it is already in smart profiles: https://splunk.github.io/splunk-connect-for-snmp/main/configuration/configuring-profiles/#smartprofile-configuration)

Reply:
check the docs: https://splunk.github.io/splunk-connect-for-snmp/develop/configuration/configuring-profiles/#conditional-profiles

Reply:
Done